### PR TITLE
Add headroom, add yarn.lock

### LINKED ISSUE #174: Use Headroom for header
## Summary
Let's add [Headroom](https://github.com/KyleAMathews/react-headroom) to use for the header.

### What's new?
Header hides while scrolling but is stickied to top when you scroll back up.

### Motivation
H E A D E R

### Additional context
This is a good and simple issue for any first time contributors but we should add this if it doesn't get taken up in a week or two.

Also, if you're working on the PR, add it to the list of dependencies in the README and link the original repo to credit the author.

### PR DESCRIPTION
**Pull request by kartikcho**
Resolves #174 and supersedes #205 

Increased width of header to 100% and added shadow 
image

Let me know if the shadow looks good.

### CODE REVIEW

**Comment by kartikcho** [comment]
I agree too partially but removing it looks kinda out of place for a header but I'm okay whatever is decided.

**Comment by naveen521kk** [comment]
I don't like the shadow as it looks little off from theme.

**Comment by kartikcho** [comment]
Done @naveen521kk !

File: src/components/header/Header.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React from "react";
+import Headroom from "react-headroom";
 import "./Header.css";
 import {Fade} from "react-reveal";
```
Comment (naveen521kk) [inline src/components/header/Header.js:4]: This import is not required.
```suggestion
```

**Comment by naveen521kk** [comment]
Also, @kartikcho add the React Headroom to the Readme file as well.

File: src/components/header/Header.js (hunk 1 above)
Comment (kartikcho) [reply src/components/header/Header.js:4]: oh, thanks!

**Pull request merged**